python/ray/serve/_private/logging_handlers.py

```diff
@@ -0,0 +1,114 @@
+import aiofiles
+import asyncio
+import logging
+
+
+class AsyncRotatingFileHandler(logging.handlers.RotatingFileHandler):
```

Comment:
That's generally not a great idea -- this makes logging unreliable and dependent on the load on the event log.

Instead i'd recommend doing the following

 - Keep writing to file as sync
 - Make heavy-weight ops (that aren't writing) async (offloading to a thread-pool)